src/nl/blog/2025/01/easy-a11y.md

```diff
@@ -0,0 +1,71 @@
+
+Met nog een vleugje CSS erbij, kun je het geheel nog mooier maken. Een skip to content link hoeft niet zichtbaar te zijn, met `position: absolute` en `top: -50px` bijvoorbeeld, kun je je link verbergen. Geen JavaScript voor nodig. Ook niet om deze zichtbaar te maken of een tab-toets te detecteren. Voeg in je CSS een regel toe voor je link met `:focus` en je link zal automagisch verschijnen zodra een gebruiker de tab-toets gebruikt.
+
+Hoe ziet het er dan nu allemaal uit? Nou zo bijvoorbeeld!
+
+```
+<body>
+  <a href="#main" class="skip-link">Skip to content</a>
+  <header>
+    <!-- Je header en navigatie -->
+  </header>
+  <main id="main">
+    <!-- Je content -->
+  </main>
+</body>
+```
+```
+.skip-link {
+  position: absolute;
+  top: -50px;
+}
+.skip-link:focus {
+  top: 10px;
+}
+```
+
+## Toegankelijke modals
+We zien ze liever niet, maar het is onvermijdelijk dat je als front-end developer dit een keer moet maken: een pop-up. Of je het nu een pop-up noemt, een modal, een dialog, een toast, of welke andere benamingen je nog meer kunt verzinnen: ze zijn niet altijd even gebruiksvriendelijk en een behoorlijke klus om toegankelijk te maken. Aan de slag met meerdere divjes, backdrops, buttons en niet te vergeten flink wat JavaScript om een focus trap toe te voegen. Of kan dit ook makkelijker?
+
+Natuurlijk! Je zult alsnog wel wat JavaScript nodig hebben, maar als je het native HTML element `dialog` gebruikt, heb je in een paar minuten een toegankelijke modal gebouwd. We beginnen met een `<dialog>` element met wat content en een `<button>` om de modal of pop-up te openen.
+
+Zowel JavaScript als je browser hebben beide al ingebouwde styling en functionaliteit om je toegankelijke pop-up te laten werken. Om de dialog te openen gebruik je de `showModal()` functie en de browser zal vervolgens je dialog in het midden van de pagina tonen met een lichte backdrop als achtergrond. Deze backdrop kun je vervolgens nog stylen met het `::backdrop` pseudo element. 
+
+Ook zal er direct een focus trap toegepast worden. Zolang je modal geopend is, kun je met de tab-toets niet bij elementen onder je backdrop komen. Laten we nu eens kijken hoe dit er dan uit ziet. Ik ga in dit voorbeeld uit van Vanilla JS, maar dit is natuurlijk in elk framework toe te passen. Herschrijf en gebruik de `openModal` en `closeModal` zoals je elke functie gebruikt in je favoriete framework.
+
+```
+<dialog>
+  <!--   Je content -->
+  <button onclick="closeModal()">Sluit modal</button>
+</dialog>
+<button onclick="openModal()">Open modal</button>
+```
+```
+const modal = document.querySelector('dialog');
+function openModal() {  
+  modal.showModal();
+}
+function closeModal() {  
+  modal.close();
+}
+```
+
+## Goede voornemens
+Het duurt nog een paar maanden voor de EAA in gaat, maar jij bent nu alvast voorbereid. En als ik vraag wat je goede voornemens zijn, dan zeg je vast: ik ga overal een skip to content link en toegankelijke modal bouwen!
```

Comment:
Wellicht hier de term "European Accessibility Act" voluit? geeft het iets meer kracht in de conclusie.

Reply:
@JoseeWouters als je dat wil, kan ik dat voor je aanpassen, maar ik merge alvast je blog! Dan staat het alvast klaar :)